src/plugins/tagfilter/tagfilter.js

```diff
@@ -0,0 +1,281 @@
+				filterGroupChkCnt = filterGroup.filter( function( o ) {
+					return o.isChecked === true;
+				} ).length,
+				filterGroupActiveFilters = [ ];
+
+			switch ( filterGroup[ 0 ].type ) {
+			case "checkbox":
+				if ( filterGroupCnt > 1 && filterGroupChkCnt > 0 ) {
+					filterGroup.forEach( function( filterItem ) {
+						if ( filterItem.isChecked ) {
+							filterGroupActiveFilters.push( filterItem.value );
+						}
+					} );
+				}
+				break;
+
+			case "radio":
+				if ( filterGroupCnt > 1 && filterGroupChkCnt > 0 ) {
+					let radioChkdItem;
+
+					for ( let i = 0; i < filterGroupCnt; i++ ) {
+						if ( filterGroup[ i ].isChecked === true ) {
+							radioChkdItem = filterGroup[ i ];
+							break;
+						}
+					}
+
+					if ( radioChkdItem.value !== "" ) {
+						filterGroupActiveFilters.push( radioChkdItem.value );
+					}
+				} else {
+					console.warn( componentName + ": Radio button groups must have a default selected value. If you want to display all items, add an option called \"All\" with an empty value." );
+				}
+				break;
+
+			case "select-one":
+				if ( filterGroup[ 0 ].value !== "" ) {
+					filterGroupActiveFilters.push( filterGroup[ 0 ].value );
+				}
+				break;
+			}
+
+			instance.activeFilters.push( filterGroupActiveFilters );
+		}
+	},
+
+	// Match tagged items to active filters and only return items that have an active filter in every filter group
+	matchItemsToFilters = function( instance ) {
+		let filtersGroups = instance.activeFilters.length;
+
+		instance.items.forEach( function( item ) {
+			let itemTags = item.tags,
+				matchCount = 0;
+
+			instance.activeFilters.forEach( function( filterGroup ) {
+				if ( filterGroup.length === 0 ) {
+					matchCount++;
+				} else {
+					let itemIncludesFilter = filterGroup.filter( function( f ) {
+						return itemTags.includes( f );
+					} ).length;
+
+					if ( itemIncludesFilter ) {
+						matchCount++;
+					}
+				}
+			} );
+
+			matchCount === filtersGroups ? item.isMatched = true : item.isMatched = false;
+		} );
+	},
+
+	// Update list of visible items according to their "isMatched" property
+	updateDOMItems = function( instance ) {
+		const updatedItemsList = instance.items.forEach( function( item ) {
+			let domItem = document.querySelector( "#" + item.id ),
+				matched = item.isMatched;
+
+			if ( matched ) {
+				if ( domItem.classList.contains( filterOutClass ) ) {
+					domItem.classList.remove( filterOutClass );
+				}
+			} else {
+				if ( !domItem.classList.contains( filterOutClass ) ) {
+					domItem.classList.add( filterOutClass );
+				}
+			}
+		} );
+
+		return updatedItemsList;
+	},
+
+	// Utility method to update stored active filters, update stored items and update visibility of tagged items
+	update = function( instance ) {
+		refineFilters( instance );
+		matchItemsToFilters( instance );
+		updateDOMItems( instance );
+	};
+
+// When a filter is updated
+$document.on( "change", selectorCtrl, function( event )  {
```

Comment:
Didn't check if we already have that but We will need to provide the instruction where the filter form interface need to be before the content being tag filtered.

Reply:
This is for "Accessibility guidance"